src/applications/disability-benefits/2346/manifest.json

```diff
@@ -3,8 +3,5 @@
   "entryFile": "./app-entry.jsx",
   "entryName": "hearing-aid-batteries-and-accessories",
   "rootUrl": "/hearing-aid-batteries-and-accessories",
-  "production": false,
-  "template": {
-    "vagovprod": false
-  }
+  "e2eTestsDisabled": true
```

Comment:
long term do we want to nest app specific attributes? I think only build stuff should be root level or maybe nested in its own grouping.

Reply:
Yeah, I think that'd be a good next iteration. :+1: